load_balancer/nginx.conf

```diff
@@ -10,15 +10,32 @@ http {
     default_type  application/json;
 
     keepalive_timeout  65;
-    
-    upstream messenger_upstream {
-      server messenger:4000;
-    }
+
     server {
         listen       80;
-        server_name  localhost;
+
         location / {
-          proxy_pass http://messenger_upstream;
+          proxy_pass http://localhost:8080;
```

Comment:
**Annotation**:
This is the directive that forwards a request to `*:80` to the handler in the other server.